roles/openshift_node_upgrade/handlers/main.yml

```diff
@@ -29,3 +29,6 @@
   when:
   - (not skip_node_svc_handlers | default(False) | bool)
   - not (node_service_status_changed | default(false) | bool)
+
+- name: reload systemd units
```

Comment:
Can we add a note here to use the ansible module when it is supported or use it if it is already supported?
http://docs.ansible.com/ansible/latest/systemd_module.html